modules/ROOT/pages/index.adoc

```diff
@@ -159,8 +159,9 @@ To upgrade to premium and increase your TPS limit, contact the MuleSoft account
 For CloudHub applications using Mule 3, Object Store v2 overrides the `_defaultUserObjectStore` object store. To use Object Store v2 in Mule 3 CloudHub applications, set the Object Store reference field to the `_defaultUserObjectStore` value.
 * Using Object Store v2 with multi-worker CloudHub applications might result in data discrepancies or key clashes.
 +
-To avoid these issues, use a distributed key-value store as a lock to synchronize access to Object Store v2.
-For information about distributed locking in Mule runtime engine, see xref:mule-runtime::distributed-locking.adoc[Distributed Locking].
+This behavior occurs because the "Distributed Locking" feature is not available in CloudHub when working with Object Store v2. As a result, under certain conditions, when multiple Mule components attempt to update Object Store v2 tables simultaneously, such calls may fail.
```

Comment:
```suggestion
This behavior occurs because the "Distributed Locking" feature isn't available in CloudHub and CloudHub 2.0 when working with Object Store v2. As a result, under certain conditions, when multiple Mule components attempt to update Object Store v2 tables simultaneously, such calls may fail.
```